### PR TITLE
Add ".php_cs" file to the repository

### PR DESCRIPTION
**Pull request by henriquemoody**
Perhaps it can be helpful.

I did run `php-cs-fixer fix`, as you can se on https://github.com/henriquemoody/phpunit/commit/5742db5, but I didn't send those changes on this pull requests because I think that @sebastianbergmann would like to do himself.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
You could just enable StyleCI on this repo and have status API integration instead of trusting contributors to run php-cs-fixer.

**Comment by sebastianbergmann** [comment]
I actually use

```
php-cs-fixer fix src --fixers=duplicate_semicolon,empty_return,extra_empty_lines,join_function,list_commas,no_blank_lines_after_class_opening,no_empty_lines_after_phpdocs,phpdoc_indent,phpdoc_no_access,phpdoc_no_empty_return,phpdoc_no_package,phpdoc_params,phpdoc_scalar,phpdoc_to_comment,phpdoc_trim,return,self_accessor,single_quote,spaces_before_semicolon,spaces_cast,ternary_spaces,trim_array_spaces,unused_use,whitespacy_lines,align_double_arrow,align_equals,concat_with_spaces
```

**Comment by henriquemoody** [comment]
I can update it. What you think?

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->files()
+    ->in('build')
+    ->in('src')
+    ->in('tests')
+    ->name('*.php')
+    ->name('*.phpt');
+
+return Symfony\CS\Config\Config::create()
+    ->level(\Symfony\CS\FixerInterface::PSR2_LEVEL)
```
Comment (GrahamCampbell) [inline .php_cs]: this wrong

Comment (GrahamCampbell) [inline .php_cs]: needs to be NONE_LEVEL

Comment (GrahamCampbell) [inline .php_cs]: or indeed symfony level, since that's the default

Comment (GrahamCampbell) [inline .php_cs]: the command @sebastianbergmann provided wouldn't be using the psr2 level

**Comment by sebastianbergmann** [comment]
@henriquemoody Yes, please.

**Comment by henriquemoody** [comment]
I made the changes, already =)

**Comment by sebastianbergmann** [comment]
Merged manually, thanks.

**Pull request closed without merging**